packages/react-integration/cypress/integration/card.spec.ts

```diff
@@ -5,12 +5,12 @@ describe('Card Demo Test', () => {
 
   it('Verify that selectable card can be selected and unselected with keyboard input', () => {
     cy.get('#selectableCard').focus();
-    cy.focused().should('have.class', 'pf-m-selectable');
```

Comment:
So non raised card can no longer be selected or selectable?

Reply:
correct - I think it's the same as disabled from my understanding

Reply:
I don't think that is the case.  We are still applying the style and we have an example for that use case.  We should be testing both here.